fairlearn/metrics/_metric_frame.py

```diff
@@ -39,6 +41,18 @@
     "Please remove non-scalar columns from your metric frame or use parameter errors='coerce'."
 
 
+def apply_to_dataframe(
```

Comment:
please add complete docstrings to methods and functions, makes the review much easier, since then I'd know what the method/function's supposed to do, and I can check the implementation against that :)

Reply:
I've been adding some. The documentation is far from final though, since the code isn't finalised yet....